src/components/AccountTab/EmptyRow.tsx

```diff
@@ -6,7 +6,7 @@ import styles from './styles.module.scss';
 const EmptyRow: React.FC<{}> = () => (
   <tr>
     <td colSpan={2} className={styles.emptyLine}>
-      No accounts have been whitelisted.
+      No accounts have been permitted.
```

Comment:
Are we using `permit` or `allow` for the properties?

Reply:
solved this by removing more words